frontend/src/components/PlayComponents/SelfCard.js

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components/macro';
+import ActionButton from '../commons/ActionButton';
+import Input from '../commons/Input';
+import { GiTwoCoins } from 'react-icons/gi';
+
+export default function SelfCard({ bet, fold, minBet, cash, active }) {
+  const [betValue, setBetValue] = useState(minBet);
+  const [betTooSmall, setBetTooSmall] = useState(false);
+  const [betToLarge, setBetTooLarge] = useState(false);
+
+  useEffect(() => {
+    setBetTooLarge(betValue > cash);
+    setBetTooSmall(betValue < minBet);
+  }, [betValue, cash, minBet]);
+
+  return (
+    <SelfCardStyled active={active}>
+      <Info>
+        <p>
+          <GiTwoCoins /> {cash}
+        </p>
+        <p>Minimum Bet: {minBet}</p>
+        <p> your guess: 97</p>
+      </Info>
+      <Actions>
+        {minBet ? (
+          <ActionButton
+            disabled={!active || minBet > cash}
+            onClick={handleCall}
+          >
+            call
+          </ActionButton>
+        ) : (
+          <ActionButton disabled={!active} onClick={handleCheck}>
+            check
+          </ActionButton>
+        )}
+        <ActionButton disabled={!active} onClick={fold}>
+          {' '}
+          fold
+        </ActionButton>
+        {betTooSmall && <p>Your Bet is too small. Please enter a larger bet</p>}
+        {betToLarge && (
+          <p> Your bet exceeds your cash. Please enter a smaller Bet</p>
+        )}
+        <form onSubmit={handleRaise}>
```

Comment:
```suggestion
        <form onSubmit={handleRaiseSubmit}>
```
Maybe make it clear that you handle an event here